### PR TITLE
Nopatch CVE-2023-23000, CVE-2023-23039, CVE-2023-26242, CVE-2023-32252, CVE-2023-32257, CVE-2023-32258, CVE-2023-33951, CVE-2023-33952, CVE-2023-37453, CVE-2023-42754, CVE-2023-42756, CVE-2023-45871, CVE-2023-4611, CVE-2023-5345

### PR DESCRIPTION
**Pull request by rlmenge**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

- Nopatch CVE-2023-23000, CVE-2023-23039, CVE-2023-26242, CVE-2023-32252, CVE-2023-32257, CVE-2023-32258, CVE-2023-33951, CVE-2023-33952, CVE-2023-37453, CVE-2023-42754, CVE-2023-42756, CVE-2023-45871, CVE-2023-4611, CVE-2023-5345

###### Change Log  

- Nopatch CVE-2023-23000, CVE-2023-23039, CVE-2023-26242, CVE-2023-32252, CVE-2023-32257, CVE-2023-32258, CVE-2023-33951, CVE-2023-33952, CVE-2023-37453, CVE-2023-42754, CVE-2023-42756, CVE-2023-45871, CVE-2023-4611, CVE-2023-5345

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2023-23000
- https://nvd.nist.gov/vuln/detail/CVE-2023-23039
- https://nvd.nist.gov/vuln/detail/CVE-2023-26242
- https://nvd.nist.gov/vuln/detail/CVE-2023-32252
- https://nvd.nist.gov/vuln/detail/CVE-2023-32257
- https://nvd.nist.gov/vuln/detail/CVE-2023-32258
- https://nvd.nist.gov/vuln/detail/CVE-2023-33951
- https://nvd.nist.gov/vuln/detail/CVE-2023-33952
- https://nvd.nist.gov/vuln/detail/CVE-2023-37453
- https://nvd.nist.gov/vuln/detail/CVE-2023-42754
- https://nvd.nist.gov/vuln/detail/CVE-2023-42756
- https://nvd.nist.gov/vuln/detail/CVE-2023-45871
- https://nvd.nist.gov/vuln/detail/CVE-2023-4611
- https://nvd.nist.gov/vuln/detail/CVE-2023-5345

### CODE REVIEW

File: toolkit/scripts/mariner-required-configs.json [hunk 1]
```diff
@@ -1156,6 +1156,62 @@
...
+            "CONFIG_SMB_SERVER": {
+                "value": [
+                    "<missing>",
+                    "is not set"
+                ],
+                "arch": [
+                    "AMD64",
+                    "ARM64"
+                ],
+                "comment": "Needs to be off for CVE-2023-32258, CVE-2023-32252 (and several others)",
```
Comment (christopherco) [inline toolkit/scripts/mariner-required-configs.json:1211]: Please annotate out what "several others" are

File: SPECS/kernel/CVE-2023-5345.nopatch [hunk 2]
```diff
@@ -0,0 +1,5 @@
+CVE-2023-5345 - introducing commit not present 5.15.135.1
+Note the file moved from fs/cifs/fs_context.c → fs/smb/client/fs_context.c
```
Comment (christopherco) [inline SPECS/kernel/CVE-2023-5345.nopatch:2]: Thanks for noting this!

**Review by christopherco — CHANGES_REQUESTED** [review]
Change in general LGTM. Please add the extra CVE numbers into the required configs comment so future devs don't need to hunt for them

**Review by christopherco — APPROVED** [review]

File: toolkit/scripts/mariner-required-configs.json (hunk 1 above)
Comment (rlmenge) [reply toolkit/scripts/mariner-required-configs.json]: Done :)

**Review by Camelron — APPROVED** [review]

**Pull request merged**